
# List of detected tags
DetectedTag[] tags

# Timstamp of the detection
time timestamp

# Return code
rc_common_msgs/ReturnCode return_code
